src/IIIF.php

```diff
@@ -147,6 +148,23 @@ public function buildProvider () {
 
     }
 
+    private function buildSeeAlso () {
+
+        return [
+            (object) [
+                "id" => $this->url . '/collections/islandora/object/' . $this->pid . '/datastream/MODS' ,
+                "type" => "Dataset",
+                "label" => [ 
+                    (object) [
+                        "en" => [ "Bibliographic Description in MODS" ]
+                    ]
+                ],
```

Comment:
@markpbaggett I'm not 100% certain, but I think we need to not have an array nested under the `label` property.

```suggestion
                "label" => (object) [
                        "en" => [ "Bibliographic Description in MODS" ]
                ],
```

This will output like:
```

  "seeAlso": [
    {
      "id": "http://localhost:8000/collections/islandora/object/rfta%3A1/datastream/MODS",
      "type": "Dataset",
      "label": {
        "en": [
          "Bibliographic Description in MODS"
        ]
      },
      "format": "application/xml",
      "profile": "http://www.loc.gov/standards/mods/v3/mods-3-5.xsd"
    }
  ],
```

as opposed to:

```

  "seeAlso": [
    {
      "id": "http://localhost:8000/collections/islandora/object/rfta%3A1/datastream/MODS",
      "type": "Dataset",
      "label": [
        {
          "en": [
            "Bibliographic Description in MODS"
          ]
        }
      ],
      "format": "application/xml",
      "profile": "http://www.loc.gov/standards/mods/v3/mods-3-5.xsd"
    }
  ],
```